examples-generated/aws/authbackendclient.yaml

```diff
@@ -8,15 +8,9 @@ metadata:
   name: example
 spec:
   forProvider:
-    accessKeySecretRef:
-      key: example-key
-      name: example-secret
-      namespace: upbound-system
-    backend: cert
-    secretKeySecretRef:
-      key: example-key
-      name: example-secret
-      namespace: upbound-system
+    identityTokenAudience: <TOKEN_AUDIENCE>
+    identityTokenTtl: <TOKEN_TTL>
+    roleArn: <AWS_ROLE_ARN>
```

Comment:
This is a new possible way to configure that (not only here, but for other backends too), but it works only in Vault Enterprise. The old one still [works fine](https://registry.terraform.io/providers/hashicorp/vault/4.3.0/docs/resources/aws_auth_backend_client#example-usage) too.

Personally, I don't like to leave the enterprise version as a default for that, but it is automatically generated. I can dismiss those changes, find a way how to generate particular examples or maybe copy the old ones on just the `examples` folder. What do you think?